.nucleus.yaml

```diff
@@ -8,21 +8,28 @@ branches:
             auto-start: true
             auto-start-from-forks: false
             merge-method: disabled # do not auto-merge; we'll do it ourselves
-            validate-optional-downstream-deps: true # temporarily test non-required downstreams
```

Comment:
This increases our overall execution cost with low value, unless we're actively doing this kind of flakiness analysis. We can turn this back on next time we want to refresh this list.